modules/my_health/catalog-info-v1.yaml

```diff
@@ -6,21 +6,22 @@ metadata:
     backstage.io/edit-url: https://github.com/department-of-veterans-affairs/platform-pst-console-ui-configuration/edit/main/backend-review-group/apis/my-health-api.yaml
     backstage.io/source-location: url:https://github.com/department-of-veterans-affairs/vets-api/tree/master/modules/my_health/
     backstage.io/view-url: https://github.com/department-of-veterans-affairs/platform-pst-console-ui-configuration/blob/main/backend-review-group/apis/my-health-api.yaml
```

Comment:
I think we should remove these references, as at least two of them refer to a file that will no longer exist.